Header header

float32 pulse
int32 strength
int32 frequency

